app/routes/index.js

```diff
@@ -1,9 +1,49 @@
 const express = require('express');
 const router = express.Router();
+const bcrypt = require("bcrypt-nodejs");
+
+const usersModel = require('../database/models/usersModel');
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
   res.render('index', { title: 'Express' });
 });
 
+router.get('/login', async function(req, res, next){
+  res.render('pages/login', {title: 'Log In'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8414579)
```suggestion
  res.render("pages/login", {title: 'Log In'});
```